helm-chart/splunk-connect-for-kubernetes/charts/splunk-kubernetes-logging/values.yaml

```diff
@@ -172,6 +172,7 @@ charEncodingUtf8: false
 #     firstline: "<regexp_to_detect_firstline_of_multiline>"
 #     flushInterval 5
 #   sourcetype: "<sourcetype_of_logs>"
+#   limitRecentlyModified: <time range>
```

Comment:
Do we provide a time range or time here?

Reply:
Fluentd documentation calls this "time range", so I used their wording. I'll probably use "time period", it would be more clear.